doc/source/default-settings.rst

```diff
@@ -13,11 +13,15 @@ General settings
     do not clutter your global configuration file. This is particularly useful
     if the library is shared with someone else (or just on a different machine)
     and you want them to have the same settings. For example, say you're sharing
-    a library with your friend Fulano. You have your library at::
+    a library with your friend Fulano. You have your library at:
+
+    .. code-block:: text
 
         ~/Documents/lib-with-fulano
 
-    and you've created a local configuration file at::
+    and you've created a local configuration file at:
+
+    .. code-block:: text
```

Comment:
Purpose of this is to stop highlighting `with` as if it's a code keyword:

![image](https://github.com/user-attachments/assets/751c6db6-3e52-40ae-aa86-24e065651eee)

Reply:
Haha, never noticed that. Thanks!

I think I'd prefer setting `highlight_language = "text"` (see the [sphinx docs](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-highlight_language)) in the configuration file for this? The implicit Python highlighting sounds like it'll just end up making more silly errors like this.

Reply:
Yes, definitely!

Reply:
Added in [56f1966](https://github.com/papis/papis/pull/1010/commits/56f19668a1fa29caad922a5fc16ece7745332bcd), could you check this is ok?

I took a surface-level look whether this breaks previous code blocks that are indeed intended to be Python, looks like generally they are still in Python. I wouldn't be surprised if we miss a couple of those, though. Of course, not having syntax highlighting in some example is not a very big deal.

Reply:
Looks good to me!

I don't recall depending on the default Python highlighting anywhere, so this should be fine. Regardless, it doesn't need to all be fixed in this PR :grin: